uiowa/uiowa
# initial commit using uids nav changes

Implementing changes from #2969 for horizontal nav changes.

# How to test

1. Check out `i2969-integrate-horizontal-nav-changes` and pull it down
2. `blt ds` or `blt dsa` the site `sandbox.uiowa.edu`
3. Run `yarn upgrade @uiowa/uids`, then `blt frontend` then `drush @sandbox.local cr`
4. navigate to `http://sandbox.local.drupal.uiowa.edu/about/menu-variants` and check that the Horizontal menus have the following:
    1. Proper `a11y` contrast for text
    2. A white active tab, unless it is `bg--white` or `bg--white--pattern--brain`, In which case they should have a Grey active tab.
    3. Yellow arrowed border at the bottom and yellow link separators, unless it is `bg--gold` or `bg--gold--pattern--brain`, in which case they should have a Grey arrowed border at the bottom and Grey link separators.
    4. Grey Bottom hover border, unless it is `bg--black` or `bg--black--pattern--brain`
